Deadly0/bull-board-docker
# Add support for specifying Redis username when creating container

Problem: The user encountered an issue where the desired tool is not compatible with the default version of Redis installed using the DOCKER OFFICIAL IMAGE from this link: https://hub.docker.com/_/redis. The issue occurs because Redis now supports having a user.

Solution: With this commit, I've added support for specifying a Redis username when creating a container.

## Changes
- Added support for specifying Redis username when creating a container

## How to Test
1. Pull changes and create a new container with the desired Redis username.
2. Verify if the specified username is working as expected with Redis instance.

## Additional Notes
Please update the documentation in case there are any breaking changes to existing user installations.
